### PR TITLE
Fixed RMS module

### PR DESCRIPTION
**Pull request by Jambls**
Updated RMS module to use for rather than accumulate to help large value handling.

### CODE REVIEW

**Comment by MHCooke** [comment]
Contains changes to readme

File: unit_tests/rms_tests.cc [hunk 1]
```diff
@@ -49,6 +49,13 @@ TEST(RMSTests, buffer) {
 
     // buffer should now be 0, 0, 0, 0, giving rms of 0
     ASSERT_EQ(rms.calc(), 0);
+
+    Audio sample{INT16_MAX, INT16_MAX, INT16_MAX, INT16_MAX};
+
+    rms.insert(sample);
+
+    // buffer should now be 0, 0, 0, 0, giving rms of 0
```
Comment (MHCooke) [inline unit_tests/rms_tests.cc:57]: comment is wrong

Comment (Jambls) [reply unit_tests/rms_tests.cc:57]: so is this one, but it doesn't keep me up at night

**Review by MHCooke — APPROVED** [review]
looks good

**Review by bear-in-the-air — APPROVED** [review]

**Pull request merged**